pkg/distro/generic/images.go

```diff
@@ -928,6 +928,80 @@ func iotSimplifiedInstallerImage(workload workload.Workload,
 	return img, nil
 }
 
+// Make an Anaconda installer boot.iso
```

Comment:
We could probably be a bit more verbose on what exactly is a `boot.iso` but that's really just docs.